sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/CombineFnsTest.java

```diff
@@ -298,9 +298,8 @@ public String apply(String input) {
     assertThat(displayData, hasDisplayItem("combineFn1", combineFn1.getClass()));
     assertThat(displayData, hasDisplayItem("combineFn2", combineFn2.getClass()));
 
-    String nsBase = DisplayDataCombineFn.class.getName();
-    assertThat(displayData, includesDisplayDataFrom(combineFn1, nsBase + "#1"));
-    assertThat(displayData, includesDisplayDataFrom(combineFn2, nsBase + "#2"));
+    assertThat(displayData, includesDisplayDataFor("combineFn1", combineFn1));
```

Comment:
This change already makes the switch. `builder.include(..)` now takes a `path` parameter rather than a `namespace` override, and `DisplayDataMatchers.includesDisplayDataFor(..)` also takes a `path` instead of `namespace`.

The Dataflow runner indeed doesn't use the `path`, but the UI currently doesn't show the `namespace` either, so there's no user-facing change.